docs/cse/records-signals-entities-insights/view-manage-entities.md

```diff
@@ -171,16 +171,16 @@ Note that:
 id,suppressed,criticality,tags,tags_to_add,tags_to_remove
 _ip-zone1-10.0.0.5,false,default,,Office-Based,
 _ip-zone1-10.0.0.6,true,default,,Office-Based,Remote
-_ip-zone1-10.0.0.7,false,Executive,Office-Based,,
```

Comment:
this didn't follow our instructions a few lines up of not including `tags` when you have `tags_to_add`: https://github.com/SumoLogic/sumologic-documentation/compare/main...davidsnyder-jask:sumologic-documentation:patch-2#diff-aba5be95cb80648467ad531397991d4df3d4843dc44d35824977e922d943e10eL152